pkgs/by-name/te/television/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "television";
+  version = "0.5.0";
+  src = fetchFromGitHub {
+    owner = "alexpasmantier";
+    repo = pname;
+    rev = "${version}";
+    hash = "sha256-yi8lPm3zkmamN6gPlGfojNlIXM1cgSr1zL2zMNni5f0=";
+  };
+  cargoHash = "sha256-1SdyVtMjkfXH9iGew9i8xpx8WlUly4vIcKY3weeW3LQ=";
+  meta = {
+    description = "Television is a blazingly fast general purpose fuzzy finder";
+    longDescription = ''
+      Television is a blazingly fast general purpose fuzzy finder TUI written
+      in Rust. It is inspired by the neovim telescope plugin and is designed
+      to be fast, efficient, simple to use and easily extensible. It is built
+      on top of tokio, ratatui and the nucleo matcher used by the helix editor.
+    '';
+    homepage = "https://github.com/alexpasmantier/television";
+    license = lib.licenses.mit;
+    mainProgram = "tv";
+    maintainers = with lib.maintainers; [ louis-thevenet ];
+  };
```

Small nit but we should add newlines between the sections like this.

```suggestion
rustPlatform.buildRustPackage rec {
  pname = "television";
  version = "0.5.0";

  src = fetchFromGitHub {
    owner = "alexpasmantier";
    repo = pname;
    rev = "${version}";
    hash = "sha256-yi8lPm3zkmamN6gPlGfojNlIXM1cgSr1zL2zMNni5f0=";
  };

  cargoHash = "sha256-1SdyVtMjkfXH9iGew9i8xpx8WlUly4vIcKY3weeW3LQ=";

  meta = {
    description = "Television is a blazingly fast general purpose fuzzy finder";
    longDescription = ''
      Television is a blazingly fast general purpose fuzzy finder TUI written
      in Rust. It is inspired by the neovim telescope plugin and is designed
      to be fast, efficient, simple to use and easily extensible. It is built
      on top of tokio, ratatui and the nucleo matcher used by the helix editor.
    '';
    homepage = "https://github.com/alexpasmantier/television";
    license = lib.licenses.mit;
    mainProgram = "tv";
    maintainers = with lib.maintainers; [ louis-thevenet ];
  };
```